src/Arcus.WebApi.Tests.Unit/Hosting/TestApiServer.cs

```diff
@@ -112,18 +120,48 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 });
             });
 
-            builder.UseStartup<TestStartup>();
+            builder.Configure(app =>
+            {
+                foreach (Action<IApplicationBuilder> configure in _configures)
+                {
+                    configure(app);
+                }
+
+                app.UseMiddleware<ExceptionHandlingMiddleware>();
+                app.UseMiddleware<TraceIdentifierMiddleware>();
```

Comment:
What is this? 🤔

Reply:
Oh, that's a nice one, that's test middleware to enable/disable the `Request.TraceIdentifier` from the HTTP request so we can see that the correlation middleware correctly generates one for us if it's not present.

Reply:
So it's our own internal stuff or not?

Reply:
Yes, test middleware to manipulate the `Request.TraceIdentifier`.